src/util/simplify_expr_int.cpp

```diff
@@ -774,15 +771,15 @@ simplify_exprt::resultt<> simplify_exprt::simplify_bitwise(const exprt &expr)
     return std::move(new_expr);
 }
 
-bool simplify_exprt::simplify_extractbit(exprt &expr)
+simplify_exprt::resultt<> simplify_exprt::simplify_extractbit(const exprt &expr)
```

Comment:
Interface should be `const extractbit_exprt &expr`

Reply:
in #4874